### PR TITLE
enh(release): Add Centreon Web 20.10.12 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

**Review by mdebauchez — APPROVED** [review]

File: fr/releases/centreon-core.md [hunk 1]
```diff
@@ -19,6 +19,13 @@ commerciales, vous pouvez vous rendre sur notre
 
 ## Centreon Web
 
+### 20.10.12
+
+#### Security fixes
+
+- Fixed session on account deletion
+
+
```
Comment (mdebauchez) [inline fr/releases/centreon-core.md:28]: There is one line break too much

**Review by mdebauchez — CHANGES_REQUESTED** [review]

**Review by mdebauchez — COMMENTED** [review]
There is one line break too much at lign 28 in french

**Review by mdebauchez — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 2]
```diff
@@ -19,6 +19,12 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.10.12
+
```
Comment (tuntoja) [inline en/releases/centreon-core.md:23]:
```suggestion

`October 5, 2021`

```

File: fr/releases/centreon-core.md [hunk 3]
```diff
@@ -19,6 +19,12 @@ commerciales, vous pouvez vous rendre sur notre
 
 ## Centreon Web
 
+### 20.10.12
+
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:23]:
```suggestion

`October 5, 2021`

```

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by tuntoja — APPROVED** [review]

**Pull request merged**